server/serverCluster.js

```diff
@@ -11,6 +11,7 @@ const authConfig = require('./auth/authConfig')
 const authApi = require('./auth/authApi')
 const apiRouter = require('./config/apiRouter')
 const WebSocketManager = require('./webSocket/webSocketManager')
+const scheduler = require('./system/scheduler')
```

Comment:
can you give to scheduler a more meaningful name? like recordPreviewCleanupScheduler or similar?